contracts/CrossChain.sol

```diff
@@ -167,6 +172,9 @@ contract CrossChain is Config, Initializable, ICrossChain {
         channelHandlerMap[PERMISSION_CHANNEL_ID] = PERMISSION_HUB;
         registeredContractChannelMap[PERMISSION_HUB][PERMISSION_CHANNEL_ID] = true;
 
+        channelHandlerMap[MULTI_MESSAGE_CHANNEL_ID] = MULTI_MESSAGE;
```

Comment:
We should not change the `initialize` function as it will not be called anymore.